docs/release-notes/winter-2019/README.md

```diff
@@ -0,0 +1,40 @@
+---
+{
+  "title": "Winter 2019 Release",
+  "title_metadata": false,
+  "layout_type": "LayoutArticle",
+  "summary": false,
+  "breadcrumbs": [
+    {
+      "text": "4.x.x Release Notes"
+    }
+  ],
+}
+---
+
+<cdr-doc-table-of-contents-shell>
+
+
+## Update Steps
+
+If you are not already on `@rei/cedar` >= 2.x.x, you will first need to [upgrade your project](../summer-2019/)) to the single-package version of Cedar.
+
+### For a Micro-Site
+
+- Update to `@rei/cedar` 4.x.x
+- Update to `@rei/febs` >= 6.0.0
+### For a Component Package
+
+- Update to `@rei/cedar` 4.x.x
+
+## New Features
+
+## Breaking Changes
+
+### CdrRating Count Bugfix
+
+We have resolved an issue with CdrRating where it treated the `count` property differently depending on whether it was passed as a string or a number. Now CdrRating will show the review text any time it receives a value for `count`, and will not show any review text if it does not receive a value. You should not need to make any changes to your application code to support this change.
```

Comment:
Is this Count Fix technically a "Breaking Change"?  Should it go under bug fix or something else?

Reply:
I'm thinking maybe have a "## Patches/Bugfixes"?

Reply:
it changes the behavior of the component, and it could potentially break a test or something like that. Had this been a regression in 3.0.0 i would agree, but this bug has been present since the beginning of this component.

Reply:
Ah that makes sense.